.github/workflows/installation-tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           brew update
           brew install git
           brew tap dbt-labs/dbt
-          brew install ${{ matrix.adapter }}
+          brew install -vd ${{ matrix.adapter }}
```

Comment:
Oh, let's drop the `d` and just keep the `v`. The `d` flag is shorthand for `--debug` and will open an interactive session on failure. I think that might hang the environment. `v` is shorthand for `--verbose` which will give us the additional log output we want. And, on that point, these logs are indeed....verbose 😄 . Hopefully they won't be truncated!

Reply:
👍  Reading the [manpage](https://docs.brew.sh/Manpage) again and you are totally right. Adjusted